src/components/EditableDropdown/EditableDropdown.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import Dropdown from '../Dropdown';
+
+
+const TextBox = styled.input`
+  margin-top: 2px;
+  padding: 6px;
+  font-size: 14px;
+  width: 240px;
+  outline-color: ${props => (props.changed ? 'orange' : 'default')};
```

Comment:
This needs to be a color from our theme not a hard-coded string. `props.theme.colors.status.error` would be my vote.